plugins/chat/assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -95,7 +95,7 @@ export default class ChatLivePane extends Component {
 
     // Technically we could keep messages to avoid re-fetching them, but
     // it's not worth the complexity for now
-    this.args.channel?.clearMessages();
+    this.args.channel?.messagesManager?.clearMessages();
```

Comment:
Maybe we create a `get #messagesManager` so we can do `this.#messagesManager` instead of this super redundant `this.args.channels.messagesManagers..`